packages/react-native-eudiw-pid/src/sd-jwt/types.ts

```diff
@@ -0,0 +1,85 @@
+type CountryCode = string;
+type EvidenceType = string;
+type VerificationAssuranceLevel = string;
+type OIDCFederationEntityStatement = unknown;
+type UnixTime = number;
+type DisclosureHashAlgorithm = "sha-256";
+
+type ObfuscatedDisclosures = { _sd: string[] };
+export type VerificationEvidence = Array<{
+  type: EvidenceType;
+  record: {
+    type: string;
+    source: {
+      organization_name: string;
+      organization_id: string;
+      country_code: CountryCode;
+    };
+  };
+}>;
+
+export type Verification = {
+  trust_framework: "eidas";
+  assurance_level: VerificationAssuranceLevel;
+};
+
+/**
+ * Data structure for the PID.
+ * It contains PID claims in plain text as well as verification data with emitter' information
+ *
+ * @see https://italia.github.io/eidas-it-wallet-docs/en/pid-data-model.html
+ */
+export type PID = {
+  verification: Verification & {
+    evidence: VerificationEvidence;
+  };
+  claims: {
+    unique_id: string;
+    given_name: string;
+    family_name: string;
+    birthdate: string;
+    place_of_birth: {
+      country: CountryCode;
+      locality: string;
+    };
+    tax_id_number: string;
+  };
+};
+
+export type SdJwt4VC = {
```

Comment:
what do you think about adding signature too?

Reply:
Are we gonna need to use signature attributes on our business logic? If yes, it makes sense indeed.
If otherwise it's just used to prove the token signature, I'll leave it un-modeled.

Reply:
> the user may not trust the verification made by the library

Do you see cases in which we should double-check the signature ourselves?

Reply:
It makes me think maybe even just for the support etc. but now I don't have a strong opinion on that. We will add it if needed